shared/chat/conversation/messages/attachment/image/index.js

```diff
@@ -78,70 +78,92 @@ class ImageAttachment extends React.PureComponent<Props, State> {
           onMouseEnter={this._onMouseEnter}
           onMouseLeave={this._onMouseLeave}
         >
-          <Kb.Text type="BodySemibold" style={styles.title}>
-            {this.props.title}
-          </Kb.Text>
           <Kb.Box
             style={Kb.iconCastPlatformStyles(
               Styles.collapseStyles([
-                styles.loading,
-                !this.state.loaded && styles.spinner,
+                styles.backgroundContainer,
                 {
-                  height: this.props.height,
-                  width: this.props.width,
+                  width: this.props.width + 6,
```

Comment:
Can we add a comment for the `+ 6`?

Reply:
Yeah the point is to make the div behind the photo just show up as a small border.

Reply:
On that note, can we either:
1. Increase the size of the container's border radius, or
2. Decrease the size of the image's border radius?

We want the spacing around the radius to be even. Currently, it gets fatter at the apex.

Reply:
My last commit I think fixes this, I did both 1 and 2.